core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -29,10 +29,31 @@ class InstallGenerator < Rails::Generators::AppBase
       none
     ]
 
-    PAYMENT_METHODS = %w[
-      paypal
-      bolt
-      none
+    PAYMENT_METHODS = [
```

Comment:
@waiting-for-dev I converted `PAYMENT_METHODS` into an array of payment method hashes. Let me know what you think!